src/components/DatePickerRange.react.js

```diff
@@ -132,6 +132,11 @@ export default class DatePickerRange extends Component {
 
   render() {
     const { setProps, fireEvent } = this.props;
+    let verticalFlag = true;
+    if (this.props.calendar_orientation === 'vertical') {
+      verticalFlag = false;
+    }
```

Comment:
this can be simplified to `const isVertical = (calendar_orientation === 'vertical')`